### PR TITLE
Rework objectstore subcommand

### PR DESCRIPTION
**Pull request by fgsch**
**NOTE:** This PR introduces breaking changes (e.g. the exposed interface is updated) requiring a new major version release.

### CODE REVIEW

**Comment by fgsch** [comment]
Tests would be updated shortly.

**Comment by fgsch** [comment]
Tests updated.

File: pkg/commands/objectstore/liststore.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ func NewListCommand(parent cmd.Registerer, globals *config.Data, data manifest.D
 		},
 		manifest: data,
 	}
-	c.CmdClause = parent.Command("list", "List Fastly object stores")
+	c.CmdClause = parent.Command("list", "List object stores")
```
Comment (Integralist) [inline pkg/commands/objectstore/liststore.go:32]: I think the file name should stay as `list.go`

File: pkg/commands/objectstorekeys/insert.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
+package objectstorekeys
```
Comment (Integralist) [inline pkg/commands/objectstorekeys/insert.go:1]: For consistency we should name this `create.go`

File: pkg/commands/objectstorekeys/insert.go [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	"io"
+
+	"github.com/fastly/cli/pkg/cmd"
+	"github.com/fastly/cli/pkg/config"
+	"github.com/fastly/cli/pkg/manifest"
+	"github.com/fastly/cli/pkg/text"
+	"github.com/fastly/go-fastly/v7/fastly"
+)
+
+// InsertCommand calls the Fastly API to insert a key into an object store.
```
Comment (Integralist) [inline pkg/commands/objectstorekeys/insert.go:13]: Similarly this should be `CreateCommand`.

File: pkg/commands/objectstorekeys/insert.go [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+// NewInsertCommand returns a usable command registered under the parent.
+func NewInsertCommand(parent cmd.Registerer, globals *config.Data, data manifest.Data) *InsertCommand {
+	c := InsertCommand{
+		Base: cmd.Base{
+			Globals: globals,
+		},
+		manifest: data,
+	}
+	c.CmdClause = parent.Command("insert", "Insert a key-value pair")
```
Comment (Integralist) [inline pkg/commands/objectstorekeys/insert.go:28]: Similarly, this should be `"create"`

File: pkg/commands/objectstore/liststore.go (hunk 1 above)
Comment (fgsch) [reply pkg/commands/objectstore/liststore.go]: Ye, missed this in my previous commit. Fixed.

File: pkg/commands/objectstorekeys/insert.go (hunk 4 above)
Comment (fgsch) [reply pkg/commands/objectstorekeys/insert.go]: Done.

File: pkg/commands/objectstorekeys/insert.go (hunk 3 above)
Comment (fgsch) [reply pkg/commands/objectstorekeys/insert.go]: Done.

**Review by Integralist — APPROVED** [review]

File: pkg/commands/objectstorekeys/insert.go (hunk 2 above)
Comment (fgsch) [reply pkg/commands/objectstorekeys/insert.go]: Done.

**Pull request merged**